content/documents/order-returns.mdx

```diff
@@ -0,0 +1,157 @@
+- Date based return policies can be enforced in your UI, and/or by using the new `SellerApprovalRules` resource
+
+![Order Return Flow](/images/knowledge-base/inline/order-returns/order-return-flow.png)
+
+## New Resource `OrderReturn`
+```
+"OrderReturn": {
+  "ID": "",
+  "OrderID": "" //required
+  "PaymentIDs": [],
+  "Status": "",
+  "RefundAmount": 0,
+  "ItemsToReturn": [],
+  "DateCreated": "2022-01-04T00:00:00.00+00:00",
+  "DateSubmitted": "2022-01-04T00:00:00.00+00:00",
+  "DateApproved": "2022-01-04T00:00:00.00+00:00",
+  "DateDeclined": "2022-01-04T00:00:00.00+00:00",
+  "DateCanceled": "2022-01-04T00:00:00.00+00:00",
+  "DateCompleted": "2022-01-04T00:00:00.00+00:00",
+  "LastUpdated": "2022-01-04T00:00:00.00+00:00",
+  "xp": {}
+}
+
+"ItemToReturn": {
+  "LineItemID": "", //required
+  "Quantity": 0,
+  "RefundAmount": 0,
+  "Comments", ""
+}
+```
+
+**Endpoints:**
+- `GET v1/orderreturns`
+- `GET v1/orderreturns/{returnID}`
+- `POST v1/orderreturns`
+- `PUT v1/orderreturns/{returnid}`
+- `PATCH v1/orderreturns/{returnid}`
+- `DELETE v1/orderreturns/{returnid}`
+- `DELETE v1/orderreturns/{returnid}/items/{lineitemid}`
+   - Creating or Updating Items can be done via POST/PUT/PATCH for OrderReturns
+- `POST v1/orderreturns/{returnid}/submit`
+- `POST v1/orderreturns/{returnid}/approve`
+- `POST v1/orderreturns/{returnid}/decline`
+- `POST v1/orderreturns/{returnid}/cancel`
+- `POST v1/orderreturns/{returnid}/complete`
+- `GET v1/orderreturns/{returnID}/approvals`
+- `GET v1/orderreturns/{returnID}/eliglbleapprovers`
+
+## New Resource `SellerApprovalRule`
+```
+"SellerApprovalRule": {
+  "ID": "",
+  "Name": "",
+  "Type": "OrderReturn" //only supported Type currently
+  "Description": "",
+  "OwnerID": "",
+  "ApprovingGroupID": "",
+  "RuleExpression": "",
+}
+```
+**Endpoints:**
+- `GET v1/approvalrules`
+- `GET v1/approvalrules/{ruleID}`
+- `POST v1/approvalrules`
+- `PUt v1/approvalrules/{ruleID}`
+- `PATCH v1/approvalrules/{ruleID}`
+- `DELETE v1/approvalrules/{ruleID}`
+
+**Notes:**
+- This resource is not available to buyer users, regardless of assigned roles
+- `OwnerID` is the ID of the organization who owns the rule. Only the Marketplace Owner can write a value other than their own organization ID. The value is used to look up the `ApprovingGroupID` and determine who is eligible to administer the rule
+- To enable date based `RuleExpressions` the rules engine now supports a new custom function `now()` which accepts an integer as a parameter
+   - For example, if you wanted any `OrderReturn` associated with an `Order` that was completed prior to 30 days ago to route for approval, you would set up an `SellerApprovalRule` with a `RuleExpression` of `Order.DateCompleted < now(-30)`
+   - Additionally, a customer can choose to enforce logic in the UI to prevent an `OrderReturn` from being initiated on Orders completed more than a specific time ago
```

Comment:
Another "just bringing this to light" - this is the first time "customer" is used in this document.  Is this referring to the buyer user as a customer? Or the entity using OrderCloud for commerce?